en/middleware/uxrce_dds.md

```diff
@@ -438,7 +439,96 @@ Each (`topic`,`type`) pairs defines:
+You can keep them if you want, without affecting your uXRCE-DDS applications.
+
+If you do choose to remove the dependencies, take care not to remove anything that is used by applications (for example, Java).
+
+#### `_rtps` targets are not needed
+
+The make targets with extension `_rtps` were used to build firmware that included client side RTPS code.
+
+The uXRCE-DDS middleware is included by default in builds for most boards, so you no longer need a special firmware to work with ROS 2. 
+To check if your board has it, look for `CONFIG_MODULES_UXRCE_DDS_CLIENT=y` in the `.px4board` file of your board.
+Those files are nested in [PX4-Autopilot/boards](https://github.com/PX4/PX4-Autopilot/tree/main/boards).
+
+If it is not present, or if it is set to `n`, then you have to clone the PX4 repo, modify the board configuration and manually [compile](../dev_setup/building_px4.md) the firmware.
+
+#### New client module and new start parameters
+
+As the client is implemented by a new PX4 module, you now have new parameters to start it.
+Take a look at the [client startup section](#starting-the-client) to learn how this is done.
+
+#### New file for setting which topics are published
+
+The list of topics that are published and subscribed for a particula firmware is now managed by the [dds_topic.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/uxrce_dds_client/dds_topics.yaml) configuration file.
+
+See [Supported uORB Messages](#supported-uorb-messages) and [DDS Topics YAML](#dds-topics-yaml) sections for more information.
+
+
+#### Topics no longer need to be synced between client and agent.
+
+The list of bridged topics between agent and client no longer needs to be synced, so the `update_px4_ros2_bridge.sh` script is no longer needed.
+
+#### Default topic naming convention has changed
+
+The topic naming format changed:
+
+- Published topics:  `/fmu/topic-name/out` (Fast-RTPS) to `/fmu/out/topic-name` (XRCE-DDS).
+- Subscribed topics: `/fmu/topic-name/in`(Fast-RTPS) to `/fmu/in/topic-name` (XRCE-DDS).
+
+You should update your application to the new convention.
+
+:::note
+You might also edit [dds_topic.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/uxrce_dds_client/dds_topics.yaml) to revert to the old convention.
+This is not recommended because it means that you would have to always use custom firmware.
+:::
+
+#### XRCE-DDS-Agent
+
+The XRCE-DDS agent is "generic" and independent of PX4: [micro-xrce-dds-agent](https://micro-xrce-dds.docs.eprosima.com/en/latest/agent.html).
+There are many ways to install it on your PC / companion computer - for more information see the [dedicated section](#micro-xrce-dds-agent-installation).
+
+#### Application-Specific Changes
+
+If you where not using ROS 2 alongside the agent, then you need to migrate to [eProsima Fast DDS](https://fast-dds.docs.eprosima.com/en/latest/index.html).
```

Comment:
I think this is about using DDS network without ROS. We don't have any new docs on this and I have no reason to believe it works. Perhaps here a link to the old docs to highlight the case we mean, and what is being replaced.

Reply:
Correct, I'm talking about that, and true, there is no docs, but I still wanted to give a hint about where to look.
```suggestion
If you where not using ROS 2 alongside the agent ([Fast DDS Interface ROS-Independent](https://docs.px4.io/v1.13/en/middleware/micrortps.html#agent-in-an-offboard-fast-dds-interface-ros-independent)), then you need to migrate to [eProsima Fast DDS](https://fast-dds.docs.eprosima.com/en/latest/index.html).
```